projects/core/src/user/occ/user.service.ts

```diff
@@ -62,7 +62,7 @@ export class OccUserService {
       .pipe(catchError((error: any) => throwError(error)));
   }
 
-  addUserAddress(userId: string, address: object) {
+  addUserAddress(userId: string, address: Address): Observable<any> {
```

Comment:
We can replace `any` with a type

Reply:
https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002/rest/v2/swagger-ui.html#!/Address/createAddressUsingPOST